api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/APSSResumoTrimestralCohortQueries.java

```diff
@@ -72,14 +124,46 @@ public CohortDefinition getA1() {
    * <p><b>Description:</b> Nº de pacientes que iniciou cuidados HIV nesta unidade sanitária durante
    * o trimestre e que receberam aconselhamento Pré-TARV no mesmo período
    *
+   * <ul>
+   *   <li>Nº de pacientes que iniciou Pré-TARV (Cuidados de HIV) [ {@link
+   *       ResumoMensalCohortQueries#getNumberOfPatientsWhoInitiatedPreTarvByEndOfPreviousMonthA1}
+   *       from Resumo Mensal only changes the period to quarterly)]
+   *   <li>And filter all patients registered in encounter “Ficha APSS&PP” (encounter_type =
+   *       ${prevencaoPositivaSeguimentoEncounterType}) who have the following conditions:
+   *       <ul>
+   *         <li>ACONSELHAMENTO PRÉ-TARV” (concept_id = 23886) with value_coded “SIM” (concept_id =
+   *             ${patientFoundYesConcept})
+   *         <li>And “encounter_datetime” Between StartDate and EndDate
+   *       </ul>
+   * </ul>
+   *
    * @return {@link CohortDefinition}
    */
   public CohortDefinition getB1() {
-    SqlCohortDefinition sqlCohortDefinition = new SqlCohortDefinition();
+    CompositionCohortDefinition cd = new CompositionCohortDefinition();
```

Comment:
I know there's a relation between this 2 but we need to see how have stuff merged while they need to be reviewed

Reply:
yes sure